packages/components/Callout/src/Callout.tsx

```diff
@@ -17,7 +17,7 @@ export const Callout = compose<ICalloutType>({
     const [targetNativeTag, setTargetNativeTag] = React.useState<number>(undefined);
 
     React.useLayoutEffect(() => {
-      if (target?.current) {
+      if (target ?.current) {
```

Comment:
?

Reply:
Not sure why this change was added. I'll unstage this